addons/knowledge/static/src/js/knowledge_controller.js

```diff
@@ -6,14 +6,15 @@ import FormController from 'web.FormController';
 import { MoveArticleToDialog } from './widgets/knowledge_dialogs.js';
 import emojis from '@mail/js/emojis';
 
-const emojisBlacklist = ['💩', '👎', '💔', '😭', '😢', '😐', '😕', '😞', '😢'];
+const emojisBlacklist = ['💩', '👎', '💔', '😭', '😢', '😐', '😕', '😞', '😢', '💀'];
```

Comment:
Why can't I have a skull, or a poop ? Do you know even Fabien likes the poop emoji ?